packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,351 +1,451 @@
     position: relative;
     width: 100%;
-    &:focus-within {
-      border-color: var(--aa-primary-color);
-      box-shadow: var(--aa-selected-color) 0 0 0 3px,
-        inset var(--aa-selected-color) 0 0 0 2px;
-      outline: currentColor none medium;
-    }
-    .aa-InputWrapperPrefix {
+    @at-root .aa-InputWrapperPrefix {
       align-items: center;
       display: flex;
       flex-shrink: 0;
-      flex-wrap: none;
       order: 1;
-      padding-right: var(--aa-spacing-half);
-      // container for search and loading icons
+      // Container for search and loading icons
       .aa-Label,
       .aa-LoadingIndicator {
         cursor: initial;
         flex-shrink: 0;
-        text-align: center;
-        width: calc(var(--aa-spacing) + var(--aa-icon-size));
+        padding: 0;
+        text-align: left;
+        width: calc(var(--aa-icon-size) + var(--aa-spacing));
         button {
           appearance: none;
           background: none;
           border: 0;
+          margin: 0;
+          padding: 2px;
         }
         svg {
-          color: var(--aa-primary-color);
-          left: 2px;
-          position: relative;
+          color: rgba(var(--aa-primary-color-rgb), 1);
+          height: auto;
+          max-height: var(--aa-input-icon-size);
           stroke-width: var(--aa-icon-stroke-width);
-          width: 20px;
+          width: var(--aa-input-icon-size);
         }
       }
     }
-    .aa-InputWrapper {
+    @at-root .aa-InputWrapper {
       order: 3;
       position: relative;
       width: 100%;
-      // input of the searchbox, where the placeholder and query appear
+      // Search box input (with placeholder and query)
       .aa-Input {
         appearance: none;
         background: none;
         border: 0;
-        color: var(--aa-text-color);
+        color: rgba(var(--aa-text-color-rgb), var(--aa-text-color-alpha));
         font: inherit;
-        height: calc(var(--aa-spacing) * 2.5);
+        height: var(--aa-search-input-height);
         width: 100%;
-        // remove native appearence
+        // Remove native appearence
         &::-webkit-search-decoration,
         &::-webkit-search-cancel-button,
         &::-webkit-search-results-button,
         &::-webkit-search-results-decoration {
           appearance: none;
         }
         &::placeholder {
-          color: var(--aa-muted-color);
+          color: rgba(var(--aa-muted-color-rgb), var(--aa-muted-color-alpha));
           opacity: 1;
         }
-        // remove focus effect as we moved it to parent wrapper
+        // Remove focus effect as we moved it to parent wrapper
         &:focus {
           border-color: none;
           box-shadow: none;
           outline: none;
         }
       }
     }
-    .aa-InputWrapperSuffix {
+    @at-root .aa-InputWrapperSuffix {
       align-items: center;
       display: flex;
       order: 4;
-      // accelerator to clear the query
+      // Accelerator to clear the query
```

Comment:
What do we mean by "accelerator"? It's not straight-forward, at least for me.

Reply:
This is a UI concept:

> Alternate methods for accomplishing a frequent action in a user interface support expert users by speeding up their interactions, without hindering novices.
>
> – https://www.nngroup.com/articles/ui-accelerators/

Reply:
Thanks, TIL!